mod/wikirate/set/all/wikirate.rb

```diff
@@ -19,6 +19,25 @@
     CGI::escape card.name
   end
 
+  def is_number? str
+    true if Float(str) rescue false
+  end
+
+  view :progress_bar do |args|
+    value = card.db_content
```

Comment:
This should be "raw_content".

Definitions:
card.db_content = value in database
card.content = db_content unless new? (in which case get from default rule)
card.raw_content = content unless structured (in which case get from structure rule)

Arguably, those names could us some improvement.  But in this case if we do "db_content", then we'll never be able to use progress bars for dynamic (structured) calculations.